tests/test_views.py

```diff
@@ -96,6 +151,74 @@ def test_400_parse_error(self):
         assert sanitise_json_error(response.data) == expected
 
 
+@pytest.mark.skipif(
+    django.VERSION < (4, 1),
+    reason="Async view support requires Django 4.1 or higher",
+)
+class ClassBasedAsyncViewIntegrationTests(TestCase):
+    def setUp(self):
+        self.view = BasicAsyncView.as_view()
+
+    def test_get_succeeds(self):
+        request = factory.get('/')
+        response = async_to_sync(self.view)(request)
```

Comment:
Should this pytest ext be added?

https://pypi.org/project/pytest-asyncio/

I'd expect more async code be added to this repo in the future.

Reply:
I'll be happy to add it if there is an agreement instead of using `async_to_sync`

Reply:
I'm personally indifferent. I tend to avoid additional dependencies when possible, so I personally support using `asgiref.async_to_sync` instead of `pytest-asyncio`. 

I don't expect this repo to have much more in terms of asyncio related tests beyond this, since we're already covering Django's CBV and FBV.

Unless Tom Christie feels otherwise, I'd say it's safe to stay as-is.